include/abb_librws/rws_resource.h

```diff
@@ -137,4 +137,21 @@ namespace abb :: rws
   };
 
   std::ostream& operator<<(std::ostream& os, RAPIDResource const& resource);
+
+  enum CFGDomain
```

Comment:
```suggestion
  enum class CFGDomain
```
C++11 introduced enum class that offers stronger typing. Otherwise it's less strict typing